pkg/result/filter.go

```diff
@@ -75,12 +75,36 @@ func FilterResult(ctx context.Context, result *types.Result, ignoreConf IgnoreCo
 	sort.Sort(types.BySeverity(filteredVulns))
 
 	result.Vulnerabilities = filteredVulns
+	result.MisconfSummary = calcMisconfSummary(result, misconfSummary, filteredMisconfs)
 	result.Misconfigurations = filteredMisconfs
-	result.MisconfSummary = misconfSummary
 
 	return nil
 }
 
+func calcMisconfSummary(result *types.Result, misconfSummary *types.MisconfSummary, filteredMisconfs []types.DetectedMisconfiguration) *types.MisconfSummary {
```

Comment:
How about just getting the number of ignored misconfigurations by policy and adding to the result?

Reply:
could you expand on it?